packages/components/header/src/Header.styles.ts

```diff
@@ -1,14 +1,13 @@
 import { css } from 'emotion';
```

Comment:
need to clarify about the border-bottom, if it should be dependent on the breadcrumbs or not.

Reply:
the bottom border should be determined by the layout component instead, based on the presence of sidebars. no sidebars no bottom border.